official/docs/node/pickups/buy_pickup.js

```diff
@@ -0,0 +1,6 @@
+const Easypost = require('@easypost/api');
+const api = new Easypost('EASYPOST_API_KEY');
+
+api.Pickup.retrieve('pickup_...').then((p) => {
+  p.buy('UPS', 'Same-day Pickup').then(console.log);
+});
```

Comment:
```suggestion
api.Pickup.retrieve('pickup_...').then((retrievePickup) => {
  retrievePickup.buy('UPS', 'Same-day Pickup').then(console.log);
});
```

These variable names could be more explicit, this also applied to other files that are using a single character as the variable names.